src/Services/QuestionService.php

```diff
@@ -4,17 +4,35 @@
 
 use Psr\Http\Message\RequestInterface;
 use QuizApp\Entities\QuestionTemplate;
+use QuizApp\Entities\QuizInstance;
 use QuizApp\Entities\TextTemplate;
+use ReallyOrm\Exceptions\NoSuchRowException;
 use ReallyOrm\Repository\RepositoryManagerInterface;
 
 class QuestionService extends AbstractService
 {
+    /**
+     * Constant for pagination.
+     */
+    private const RESULTS_PER_PAGE = 10;
+
     private $repositoryManager;
 
     public function setRepositoryManager(RepositoryManagerInterface $repositoryManager)
     {
         $this->repositoryManager = $repositoryManager;
     }
+    /**
+     * Counts how many user entities the Repository has.
```

Comment:
I changed this and it will be present in the next commit

Reply:
return type